src/components/DopplerPlus/PlanCalculator.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { Slider } from '../shared/Slider/Slider';
+
+const PlanCalculator = () => {
+  // TODO: get data from a double service
+  const [state, setState] = useState({ price: 15 });
```

Comment:
What do you think about simply:

```suggestion
  const [price, setPrice] = useState(15);
```

Reply:
Yes. maybe I will have to mange other values in the state in the future, but I'm not sure about it, so I think that for the moment it's a nice change.

Reply:
Right, I am not sure if having more than an independent value in the same "state" is a good practice. And, when the values are not independent, I am pretty sure that using `useReducer` is the best approach.

Reply:
I mean, maybe I could manage a couple of values like: { price: 15, amount: 1500 }. I should apply this logic now maybe.

Reply:
in that case, probably using a reducer is a better option than a complex value with `usesState`